_layouts/home.html

```diff
@@ -23,7 +23,7 @@ <h3 class="sites-tested"></h3>
     <div class="description">
       <h2>Firefox strikes back!</h2>
       <p>Firefox Quantum is soon to be launched and this is our opportunity to make Firefox rise again. Participate in the Firefox Quantum Sprint and make a difference by ensuring that Firefox runs smoothly in your region.</p>
-      <iframe width="560" height="315" src="https://www.youtube.com/embed/ck6bt3VAqwA?showinfo=0" frameborder="0" allowfullscreen></iframe>
+      <<iframe src="https://air.mozilla.org/desktop-webcompat-short-version/video/" width="640" height="380" frameborder="0" allowfullscreen></iframe>
```

Comment:
1) There is a < too much
2) Can we link to the youtube version of it like it was before? Otherwise we need to change the width and height of the iframe and also somehow get rid of the sentence below it

![screen shot 2017-10-04 at 19 29 32](https://user-images.githubusercontent.com/330324/31190083-60c42ba6-a93a-11e7-82b1-be53811d86b5.png)

![screen shot 2017-10-04 at 19 29 35](https://user-images.githubusercontent.com/330324/31190086-65149ec0-a93a-11e7-8512-4e4361ebdc17.png)